test/cppsuite/test_harness/workload/thread_context.h

```diff
@@ -124,24 +128,25 @@ class thread_context {
     /*
      * Generic update function, takes a collection_id and key, will generate the value.
      *
-     * Returns true if it successfully updates the key, false if it receives rollback from the API.
+     * Returns a boolean indicating if a rollback is required.
```

Comment:
We need to say the value of the boolean to avoid confusion.
`Returns true if...`